Makefile

```
@@ -53,7 +56,9 @@ generate-rosa-brand-logo:
 generate-hive-templates: generate-oauth-templates
 	if [ -z ${IN_CONTAINER} ]; then \
 		$(CONTAINER_ENGINE) pull quay.io/app-sre/python:3 && $(CONTAINER_ENGINE) tag quay.io/app-sre/python:3 python:3 || true; \
+		$(CONTAINER_ENGINE) run $(CONTAINER_RUN_FLAGS) python:3 /bin/sh -c "wget -nc https://github.com/stolostron/policy-generator-plugin/releases/download/v1.9.1/linux-amd64-PolicyGenerator; chmod +x linux-amd64-PolicyGenerator ; PolicyGenerator=$(shell pwd)/linux-amd64-PolicyGenerator ${GEN_POLICY}";\
```

Updates to not download to local filesystem by using `/tmp` and set `-u` to get correct file premissions in generated output.

```suggestion
		$(CONTAINER_ENGINE) run $(CONTAINER_RUN_FLAGS) -u `id -u ${USER}`:`id -g ${USER}` python:3 /bin/sh -c "wget -nc https://github.com/stolostron/policy-generator-plugin/releases/download/v1.9.1/linux-amd64-PolicyGenerator -O /tmp/linux-amd64-PolicyGenerator; chmod +x /tmp/linux-amd64-PolicyGenerator ; PolicyGenerator=/tmp/linux-amd64-PolicyGenerator ${GEN_POLICY}";\
```

---

Used this `--user $(id -u):$(id -g)` and it seems to work now. i'm not 100% sure it's the same as `-u `id -u ${USER}`:`id -g ${USER}`` in the suggetion